conda/testing/integration.py

```diff
@@ -203,21 +226,8 @@ class Commands:
     RUN = "run"
 
 
-@deprecated("23.9", "24.3", addendum="Use `monkeypatch.chdir` instead.")
-@contextmanager
-def temp_chdir(target_dir):
-    curdir = os.getcwd()
-    if not target_dir:
-        target_dir = curdir
-    try:
-        os.chdir(target_dir)
-        yield
-    finally:
-        os.chdir(curdir)
-
-
-@deprecated("23.9", "24.3", addendum="Use `conda.testing.conda_cli` instead.")
-def run_command(command, prefix, *arguments, **kwargs):
+@deprecated("23.9", "25.3", addendum="Use `conda.testing.conda_cli` instead.")
```

Comment:
Postponed since `run_command` is still called from helper functions that had not been marked for deprecation yet